build.assets/tooling/cmd/difftest/ast_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseAST(t *testing.T) {
+	methods, err := parseMethodMap("_fixtures/ast/head/a_simple_test.go", nil)
```

Comment:
I'd prefer to use `filepath.Join` here instead of hard coding the `/` separator.

It probably doesn't really matter for this tool, but I'm always on the lookout for things like this because there will be a lot of fixes necessary when we try to run Teleport on Windows.